app/importers/activity_insight_importer.rb

```diff
@@ -202,17 +202,9 @@ def call
           end
 
           activity_insight_file_location = pub.postprints&.first&.location
-          aif = pub_record.activity_insight_oa_files.find_by(location: activity_insight_file_location)
```

Comment:
This line will be used for the export oa status code to help get the intellcont_id & post_file_id so we probably don't want to delete it yet.